server/game/cards/06-WoE/ShrewdInvestor.js

```diff
@@ -0,0 +1,23 @@
+const Card = require('../../Card.js');
+
+class ShrewInvestor extends Card {
```

Comment:
Class name should be `ShrewdInvestor`

Reply:
Oops, good catch.  My fingers have problems with that word for some reason.  Fixed!